src/model/filterUnsafeBlockElements.ts

```diff
@@ -0,0 +1,32 @@
+export const filterUnsafeBlockElements = (data: CAPIType): CAPIType => {
+    const elements: CAPIElement[] = data.blocks[0].elements.reduce(
+        (acc: CAPIElement[], cur: CAPIElement) => {
+            if (
+                cur._type ===
+                    'model.dotcomrendering.pageElements.EmbedBlockElement' &&
+                !cur.safe
+            ) {
```

Comment:
we could explore `frontend` doing this for us, rather us having to filter out these elements ourself

Reply:
+1

Reply:
I think I'm missing something - why create a new BlockElement type rather than just branching `EmbedBlockElement`  react component into `SafeEmbedBlockElement` and `UnsafeEmbedBlockElement`?

Reply:
good point, this was because the workaround I used to set the high was done using react hooks, and in order to get hooks to work we needed to hydrate that component on the client. Therefore I thought i would be simpler to split out `SafeEmbedBlockElement`. Though the changes @gtrufitt recommended means we wont need to do this